openai-core/src/commonMain/kotlin/com.aallam.openai.api/run/RunStepDetails.kt

```diff
@@ -89,9 +89,18 @@ public sealed interface ToolCallStep {
     @SerialName("function")
     public data class FunctionTool(
         /**
-         * The name of the function.
+         * The ID of the tool call object.
          */
-        @SerialName("name") public val name: String,
+        @SerialName("id") public val id: String,
```

Comment:
I just noticed that all `ToolCallStep` types have an `id`, maybe we can have a value class named `ToolCallStepId` for them!